docs/_guide/styles.md

```diff
@@ -160,6 +210,10 @@ An element can apply styles to itself with the `:host` and `:host()` CSS psuedo-
 
 {% include project.html folder="docs/style/hostselector" openFile="my-element.js" %}
 
+COMPLETE NONSENSE, CORRECTED IN SEPARATE PR 
+
+------ start nonsense -----
+
```

Comment:
I think the specificity bit is sensical, and important :)

Reply:
it's wrong tho, think i got it backwards